census/src/test/java/io/grpc/census/CensusModulesTest.java

```diff
@@ -1138,9 +1141,11 @@ public void convertToTracingStatus() {
   @Test
   public void generateTraceSpanName() {
     assertEquals(
-        "Sent.io.grpc.Foo", CensusTracingModule.generateTraceSpanName(false, "io.grpc/Foo"));
+        "Sent.io.grpc.Foo", CensusTracingModule
```

Comment:
Revert whitespace changes? Here and on the next line.

Reply:
Done.